apps/lcs/behaviours/custom-redirect.js

```diff
@@ -1,9 +1,16 @@
+const { shouldRedirectToBefore1988 } = require('../../../utils/index.js');
+
 module.exports = superclass => class extends superclass {
   successHandler(req, res, next) {
     const currentRoute = req.form.options.route;
     const action = req.params.action;
 
     if (req.sessionModel.get('isCurrentTenant') && currentRoute === '/tenant-address' && action === 'edit') {
+      if(shouldRedirectToBefore1988(req.sessionModel.get('tenant-dob')) &&
+       !req.sessionModel.get('steps').includes('/before-1988')) {
+        this.emit('complete', req, res);
+        return res.redirect('/before-1988');
+      }
```

Comment:
this should be extracted to separate statement specifically for `/tenant-details` route, 
`if (currentRoute === '/tenant-details' && action === 'edit') {`
and then `continueOnEdit` can be removed from `/tenant-details` step in index.js file